src/app/views/PublishesNotices.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import {formatTimestamp, thunks, type Thunk} from "@welshman/app"
+  import {ctx} from "@welshman/lib"
+  import {type Connection} from "@welshman/net"
+  import {sortBy} from "ramda"
+  import {get} from "svelte/store"
+  import {fly} from "svelte/transition"
+  import AltColor from "src/partials/AltColor.svelte"
+  import Input from "src/partials/Input.svelte"
+  import ThunkNotice from "src/partials/ThunkNotice.svelte"
+  import {subscriptionNotices, type PublishNotice} from "src/domain/connection"
+
+  export let search: string = ""
+
+  $: subNotices = Array.from($subscriptionNotices.values())
+    .flatMap(n => n)
+    .filter(
+      n =>
+        n.url.toLowerCase().includes(search.toLowerCase()) ||
+        n.notice.some(n => String(n).toLowerCase().includes(search.toLowerCase())),
+    )
```

Comment:
Use `createSearch` from welshman/app. Also, let's match against the actual text that is displayed to avoid unexpected false negatives. That'll require deriving all items with display information, then feeding that into search.